source/server.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const express = require('express');
+const app = express();
+require('dotenv').config();
+
+const mongoose = require('mongoose');
+mongoose.connect(MONGODB_URI, { useNewUrlParser: true, useUnifiedTopology: true });
```

Comment:
This is causing issues with your application to run, it should be:
`mongoose.connect(process.env.MONGODB_URI, { useNewUrlParser: true, useUnifiedTopology: true });`